src/Component/Metadata/Section.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Resource\Metadata;
+
+#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::IS_REPEATABLE)]
+final class Section
+{
+    private ?Operations $operations;
+
+    public function __construct(
+        private ?string $name = null,
+        private ?string $routePrefix = null,
```

Comment:
If we rename the class to `Routing`, `route` prefix here seems to be tautology 
```suggestion
        private ?string $prefix = null,
```